src/css/styles.scss

```diff
@@ -1,6 +1,42 @@
 @import 'variables';
 @import 'mixins.scss';
 
+@mixin inputFields {
+  margin: auto;
+  color: $header_color;
+  margin-top: 10px;
+  text-align: left;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+@mixin inputBoxes {
+  width: 100%;
+  font-size: 1.2em;
+  padding: 5px;
+  opacity: 1;
+}
+
+@mixin submitButtons {
+  font-size: 1.3em;
+  color: white;
+  background-color: #01579B;
```

Comment:
not sure if our variable colors would work here either, but it would be cool to try and utilize those within the mixins as well.